### PR TITLE
readme: update list of token standards

### PR DESCRIPTION
**Pull request by robert-zaremba**
* added nep-393 to the list.
* nep-141 token concept was mainly driven by my design (as documented in the past PRs), so adding myself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,28 +10,28 @@ Changes to the protocol specification and standards are called NEAR Enhancement
...
+| [0245](https://github.com/near/NEPs/blob/master/neps/nep-0245.md) | Multi Token Standard                     | @zcstarr @riqi @jriemann @marcos.sun         | Review     |
+| [0264](https://github.com/near/NEPs/blob/master/neps/nep-0264.md) | Promise Gas Weights                      | @austinabell                                 | Final      |
+| [0297](https://github.com/near/NEPs/blob/master/neps/nep-0297.md) | Events Standard                          | @telezhnaya                                  | Final      |
+| [0330](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) | Source Metadata                          | @BenKurrek                                   | Review     |
+| [0366](https://github.com/near/NEPs/blob/master/neps/nep-0366.md) | Meta Transactions                        | @ilblackdragon @e-uleyskiy @fadeevab         | Draft      |
+| [0393](https://github.com/near/NEPs/blob/master/neps/nep-0363.md) | Sould Bound Token (SBT)                  | @robert-zaremba                              | Draft      |
```
Comment (fadeevab) [inline README.md:30]:
```suggestion
| [0363](https://github.com/near/NEPs/blob/master/neps/nep-0363.md) | Sould Bound Token (SBT)                  | @robert-zaremba                              | Review      |
```

File: README.md [hunk 2]
```diff
@@ -10,28 +10,28 @@ Changes to the protocol specification and standards are called NEAR Enhancement
...
+| [0199](https://github.com/near/NEPs/blob/master/neps/nep-0199.md) | Non Fungible Token Royalties and Payouts | @thor314 @mattlockyer                        | Final      |
+| [0245](https://github.com/near/NEPs/blob/master/neps/nep-0245.md) | Multi Token Standard                     | @zcstarr @riqi @jriemann @marcos.sun         | Review     |
+| [0264](https://github.com/near/NEPs/blob/master/neps/nep-0264.md) | Promise Gas Weights                      | @austinabell                                 | Final      |
+| [0297](https://github.com/near/NEPs/blob/master/neps/nep-0297.md) | Events Standard                          | @telezhnaya                                  | Final      |
+| [0330](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) | Source Metadata                          | @BenKurrek                                   | Review     |
+| [0366](https://github.com/near/NEPs/blob/master/neps/nep-0366.md) | Meta Transactions                        | @ilblackdragon @e-uleyskiy @fadeevab         | Draft      |
```
Comment (fadeevab) [inline README.md:29]:
```suggestion
| [0366](https://github.com/near/NEPs/blob/master/neps/nep-0366.md) | Meta Transactions                        | @ilblackdragon @e-uleyskiy @fadeevab         | Final      |
```

**Review by fadeevab — COMMENTED** [review]
I suggested 2 changes, particularly, NEP-366 has been already implemented, and there is a typo in the SBT line,  and I think it's in the review stage at least.

**Comment by fadeevab** [comment]
@robert-zaremba Hey! :) Will you continue with this PR?

**Comment by robert-zaremba** [comment]
@fadeevab The PR is done, waiting for reviews.

**Comment by fadeevab** [comment]
> @fadeevab The PR is done, waiting for reviews.

You probably missed my reviews: https://github.com/near/NEPs/pull/537#discussion_r1527259803, https://github.com/near/NEPs/pull/537#discussion_r1527259920

**Review by fadeevab — APPROVED** [review]

**Review by frol — APPROVED** [review]

**Pull request merged**